test/index.coffee

```diff
@@ -24,3 +26,27 @@ Amen.describe "Haiku9 static-site generation", (context) ->
 
     context.test "Compile CoffeeScript files", ->
       assert yield exists "build/site.js"
+
+  context.test "Run the serve task", ->
+    context.test "Starts a local server on the default port", ->
+      command = join __dirname, "..", "bin", "h9"
+      child = childProcess.spawn(command, ["serve"])
+
+      # Give the server up to 5 seconds to spin up (but resolve immediately when you can)
+      start = now = new Date()
+      while now - start < 5000
```

Comment:
Any opinions on this approach? Is there a `fairmont`-ish way to do this? I wanted to be able to wait for the server to spin up, but not have an arbitrary `sleep` time in there. So this will attempt to hit the server until either a) it gets a response, or b) 5 seconds passes.